### PR TITLE
Notify slack on complete.

### PR DESCRIPTION
**Pull request by jmcarp**
Since FEC won't have access to concourse, configure the pipeline to notify slack on success or failure.

cc @ccostino

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
     - get: pipeline-tasks
     - get: terraform-templates
       trigger: true
-  - task: apply-terraform
+  - task: terraform-apply
```
Comment (ccostino) [inline pipeline.yml:8]: Did this change just to match the job name above and/or the script name below?  Trying to recall what this is in reference to.

File: credentials.example.yml [hunk 2]
```diff
@@ -10,3 +10,8 @@ rds-staging-username:
 rds-staging-password:
 rds-development-username:
 rds-development-password:
+
+slack-webhook-url:
+slack-channel:
+slack-username:
+slack-icon-url:
```
Comment (ccostino) [inline credentials.example.yml:17]: We need to share this information with folks like @erik-burgess just like the RDS credentials, right?

**Review by ccostino — COMMENTED** [review]
Thanks, @jmcarp!  Just a couple of quick questions before I merge.

File: pipeline.yml (hunk 1 above)
Comment (jmcarp) [reply pipeline.yml:8]: Yep, just renamed for consistency.

File: credentials.example.yml (hunk 2 above)
Comment (jmcarp) [reply credentials.example.yml:17]: Right!

**Review by ccostino — APPROVED** [review]
Thanks, @jmcarp!

**Pull request merged**